functions.php

```diff
@@ -14,11 +14,61 @@ function getData($db)
     return $plants;
 }
 
-function processData($plants)
-{
+
+
+function processData(array $plants):string {
     $plantRow = '';
     foreach ($plants as $plant) {
-        $plantRow .= '<div class="plant">' . $plant['id'] . $plant['common_name'] . $plant['latin_name'] . $plant['identification_image'] . $plant['canopy_level'] . $plant['life_span'];
+        $plantRow .=
+            '<div class="row"><ul><li>' .
```

Comment:
If you can put the operator '.' at the start of each row that makes it much easier to read